src/components/loader.component.tsx

```diff
@@ -0,0 +1,9 @@
+import { VNode } from '@cycle/dom';
+
+export function LoaderComponent(): VNode {
+    return (
+        <div className="loader">
+            <div className="spinner"></div>
+        </div>
+    );
+}
```

Comment:
Let's be consistent across whole project about EOF.
Also thanks for moving it to separate component 👍